### DISCUSSION TITLE
Keyboard shortcuts (like Trello)

### DISCUSSION THREAD

**Discussion by jj-zhu** [mgmeyers/obsidian-kanban#240] [Ideas & Feature Requests]
I found the keyboard shortcuts in Trello (https://trello.com/shortcuts) extremely handy when working with large boards. That is in fact a major reason I stayed with Trello instead of migrating to Notion. Would love to see obsidian-kanban have something like that.

Thanks for the good work :)

**Comment by adamalbrecht** [comment]
Same - I tried to use Obsidian Kanban (and like the idea), but the lack of keyboard shortcuts prevented it from sticking.
